src/redux/reducers/userReducer.js

```diff
@@ -40,5 +42,14 @@ export default createReducer(intialState, {
     error:
       (payload && payload.full_messages && payload.full_messages[0]) ||
       'An error occured. Please try again.'
+  }),
+  [logoutSuccess]: () => ({
+    intialState
+  }),
+  [logoutError]: (state, { payload }) => ({
+    ...state,
+    error:
+      (payload && payload.full_messages && payload.full_messages[0]) ||
+      'An error occured. Please try again.'
```

Comment:
You can use the optional chaining operator: https://v8.dev/features/optional-chaining
```suggestion
      payload?.full_messages?.[0] || 'An error occurred. Please try again.' 
```

Reply:
Nice! I didn't know that `?.[]` exists. Thanks by the extra info link